server/src/main/scala/controllers/Application.scala

```diff
@@ -327,6 +327,7 @@ trait SystemActors {
       case "LHR" => config.getString("lhr.forecast_path")
         .map(path => createForecastLHRFeed(path))
         .getOrElse(forecastNoOp)
+      case "BHX" => BHXFeed(forecast = true)
```

Comment:
We have had a discussion about using boolean flags for changing the purpose of functions previously and decided it was probably a bad practice https://martinfowler.com/bliki/FlagArgument.html

Could you rather make a BHXForecastFeed and BHXLiveFeed, then you can reuse the relevant functions with each of them.